src/include/cephfs/sharedtypes.h

```diff
@@ -0,0 +1,946 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Missing LICENSE/copyright notice.

Reply:
It didn't exist in mdstypes.h, therefore I'm not sure what note to add. Could you please share the LICENSE/copyright notice?

Reply:
Please refer to `client/RWRef.h`, this was what I did recently.

Reply:
Okay